### DISCUSSION TITLE
Magpie全屏非常好用，但能否仅缩放窗口？

### DISCUSSION THREAD

**Discussion by sora987** [Blinue/Magpie#424] [Q&A] [answered]
最近在4K显示器上玩1200x800分辨率的游戏，游戏窗口模式只能原生分辨率，如果使用magpie的2倍缩放则是全屏+黑边。

求问Magpie能不能仅2倍放大窗口而不产生黑边？

感谢！

**Comment by Blinue** [comment, marked as answer]
你是想[窗口化](https://github.com/Blinue/Magpie/issues/135)吗？

**Reply by sora987** [reply]
感谢大佬回复，看了135后，明白了窗口化难度大，那么是否可以让2倍/3倍放大的黑边消失显示背景，窗口依然强行居中？

如果难度和窗口化一样大，还请大佬无视我...

**Reply by Blinue** [reply]
窗口化功能已在计划中。有建议的话在 #135 回复即可

**Reply by MinaseInorin** [reply]
> 窗口化功能已在计划中。有建议的话在 #135 回复即可

我跟楼主情况有点像，不论用UL还是VL放大1次，都是全屏+黑边（出现在720P游戏），是现在都需要叠2次？以前0.9.1一次UL就OK

**Reply by Blinue** [reply]
> 我跟楼主情况有点像，不论用UL还是VL放大1次，都是全屏+黑边（出现在720P游戏），是现在都需要叠2次？以前0.9.1一次UL就OK

https://github.com/Blinue/Magpie/issues/448#issuecomment-1308045339
